src/ui/fragments/add/AddFormContainer.js

```diff
@@ -1,22 +1,26 @@
 import { connect } from 'react-redux';
 import FragmentForm from 'ui/fragments/common/FragmentForm';
 import { sendPostFragment } from 'state/fragments/actions';
-import { submit } from 'redux-form';
+import { DEFAULT_FORM_VALUES } from 'state/fragments/const';
 import { withRouter } from 'react-router-dom';
 import { routeConverter } from '@entando/utils';
 import { setVisibleModal } from 'state/modal/actions';
 import { ROUTE_FRAGMENT_LIST } from 'app-init/router';
 import { ConfirmCancelModalID } from 'ui/common/cancel-modal/ConfirmCancelModal';
 
+export const mapStateToProps = () => ({
+  initialValues: DEFAULT_FORM_VALUES,
+});
+
 export const mapDispatchToProps = (dispatch, { history }) => ({
   onSubmit: (fragment, saveType) => dispatch(sendPostFragment(fragment, saveType)),
-  onSave: () => { dispatch(setVisibleModal('')); dispatch(submit('fragment')); },
+  onHideCancelModal: () => { dispatch(setVisibleModal('')); },
```

Comment:
is this method used after `onCancel` below is triggered and cancel is confirmed?

Reply:
if not and its used to just hide the modal maybe naming is not really good? "cancel" should not be the part as its somehow misguiding, since we just hide the modal during save.

Reply:
I'll just rename it to `onHideModal`